tv_apps/smart-home/js/home.js

```diff
@@ -130,9 +157,71 @@
       });
     },
 
-    onCardInserted: function(card, idx) {
-      this.cardScrollable.insertNodeBefore(this._createCardNode(card), idx);
+    _onPinInit: function(newCardButtonElem) {
+      newCardButtonElem.classList.add('new-card');
+      newCardButtonElem.classList.add('new-card-transition');
+
+      // add sliding transition to card-list and every existing cards
+      var i;
+      var cardElem;
+      for(i = 0; i < this.cardListElem.children.length; i++) {
+        cardElem = this.cardListElem.children[i];
+        cardElem.classList.add('card-slide');
```

Comment:
Remove `card-slide` and just change the transition time of card from `ease 0.3` to the `bezier` specified by UX.